### PR TITLE
Fix link instructions

### PR DESCRIPTION
**Pull request by thomasw04**
Just a fix for the the link cmd

### CODE REVIEW

**Review by felixhauptmann — CHANGES_REQUESTED** [review]
That's not right... the tests are usually in `/test/pgdp` (package pgdp) or `/test`

Therefore the test sourceDir is `test`

**Comment by MaximilianAnzinger** [comment]
Tell me once both of you agree and I'll accept the PR immediately.

**Comment by thomasw04** [comment]
Yes but thats not how the windows command works... Basically in windows the arguments are switched and /test/pgdp does not mean that the folder will be placed in pgdp it does mean that the new folder is called pgdp and a test. (test folder is already present because of gradle

File: README.adoc [hunk 1]
```diff
@@ -89,14 +89,14 @@ As this is a student-driven project please keep in mind that mistakes can happen
 This has to be executed in the command line, run as administrator:
 [source,shell]
 ----
-$ mklink /d \path\to\repository\aufgabe\test \path\to\repository\
+$ mklink /d \path\to\repository\test\pgdp \path\to\repository\aufgabe\test
```
Comment (felixhauptmann) [inline README.adoc:92]: What about that?

```suggestion
$ mklink /d \path\to\artemis-repository\test \path\to\test-repository\aufgabe\test
```

File: README.adoc [hunk 2]
```diff
@@ -89,14 +89,14 @@ As this is a student-driven project please keep in mind that mistakes can happen
...
-$ mklink /d \path\to\repository\aufgabe\test \path\to\repository\
+$ mklink /d \path\to\repository\test\pgdp \path\to\repository\aufgabe\test 
 ----
 
 *Linux/MacOS:*
 
 [source,shell]
 ----
-$ ln -s /path/to/repository/aufgabe/test /path/to/repository/
+$ ln -s /path/to/repository/aufgabe/test/pgdp /path/to/repository/test
```
Comment (felixhauptmann) [inline README.adoc:99]: Same for linux...
```suggestion
$ ln -s /path/to/test-repository/aufgabe/test /path/to/artemis-repository/test
```

**Review by felixhauptmann — CHANGES_REQUESTED** [review]
I know how mklink works... it's just that you shouldnt link to a package...

**Comment by thomasw04** [comment]
You have to, because the test folder already exists because of the gradle script. Because of that the command will fail if you try to link against the test folder.

**Comment by felixhauptmann** [comment]
Your'e totally right. I missed that. but then we have to link `/path/to/test-repository/aufgabe/test/pgdp` to `/path/to/artemis-repository/test/pgdp`

**Comment by thomasw04** [comment]
@MaximilianAnzinger We are done :)

**Comment by MaximilianAnzinger** [comment]
[🐧](https://emojipedia.org/penguin/)

**Pull request merged**